packages/linear-progress/index.tsx

```diff
@@ -124,43 +155,27 @@ class LinearProgress extends React.Component {
       ...otherProps
     } = this.props;
     return (
-      <Tag className={this.classes} role="progressbar" {...otherProps}>
-        {bufferingDots && <div className="mdc-linear-progress__buffering-dots"></div>}
-        <div className="mdc-linear-progress__buffer" ref={this.bufferElement_}></div>
+      // @ts-ignore
```

Comment:
Comment on why this ignore is needed. 

Also, why do we need to destructure all of the props here if most of them are unused ?

Reply:
they need to be defined so that they don't appear in ...otherProps. It is like black listing